packaging/windows/get-convert-licenses.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+function txt_to_rtf() {
+    INPUT="$1"
+    OUTPUT="$2"
+
+    echo "{\rtf1\ansi\deff0 {\fonttbl {\f0 Times New Roman;}}" > "$OUTPUT"
+    echo "\paperh15840 \paperw12240" >> "$OUTPUT"
+    echo "\margl720 \margr720 \margt720 \margb720" >> "$OUTPUT"
+    echo "\f0\fs24" >> "$OUTPUT"
```

Comment:
Assuming that the `\` usage here is part of the RTF data, all of these strings need to be single-quoted, not double-quoted to ensure the backslashes are not interpreted as escapes.